Dockerfile

```diff
@@ -1,34 +1,41 @@
 FROM        python:3-alpine3.7
 
-ADD         requirements.txt /app/
 WORKDIR     /app
 
+EXPOSE      80
+ENV         FLASK_APP "manage.py"
+ENV         FLASK_CONFIG "production"
+
+# Setup deps
 RUN apk update && apk add py3-psycopg2 musl-dev \
     nginx supervisor git \
     openssl ca-certificates \
     gcc postgresql-dev \
  && pip install --upgrade pip
 
-RUN         pip install -r /app/requirements.txt
-COPY        dataservice docs .git migrations bin manage.py config.py setup.py /app/
-RUN         python /app/setup.py install
-
-EXPOSE      80
-ENV         FLASK_APP "manage.py"
-ENV         FLASK_CONFIG "production"
-
 # Setup nginx
-RUN         mkdir -p /run/nginx
-RUN         mkdir -p /etc/nginx/sites-available
-RUN         mkdir /etc/nginx/sites-enabled
-RUN         rm -f /etc/nginx/sites-enabled/default
-RUN         rm -f /etc/nginx/conf.d/default.conf
+RUN         mkdir -p /run/nginx && \
+            mkdir -p /etc/nginx/sites-available && \
+            mkdir /etc/nginx/sites-enabled && \
+            rm -f /etc/nginx/sites-enabled/default && \
+            rm -f /etc/nginx/conf.d/default.conf
 COPY        bin/nginx.conf /etc/nginx/nginx.conf
 
 # Setup supervisord
 RUN         mkdir -p /var/log/supervisor/conf.d
 COPY        bin/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
-#COPY        bin/gunicorn.conf /etc/supervisor/conf.d/gunicorn.conf
+
+# Setup dataservice
+COPY        requirements.txt /app/
+RUN         pip install -r /app/requirements.txt
+
+COPY        manage.py setup.py config.py /app/
+COPY        bin bin
+COPY        docs docs
+COPY        migrations migrations
+COPY        dataservice  dataservice
+COPY        .git .git
```

Comment:
@dankolbman btw, why do we need .git?

Reply:
We return the current commit hash on the status page

Reply:
Oh thats right...

Copying the entire .git folder seems like a lot just to get the commit hash for the release. We could probably write the commit hash to the package init or somewhere so that you don't need to copy the .git folder over. Just read the commit hash from the file. But that's for another PR

Reply:
yeah we should do that (calls to git are slow anyway)